### PR TITLE
search bar airbnb

### PR DESCRIPTION
**Pull request by yevheniia-biliaieva**
https://yevheniia-biliaieva.github.io/layout_search-bar-airbnb/
https://yevheniia-biliaieva.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+html {
+  box-sizing: border-box;
```
Comment (etojeDenys) [inline src/style.css:14]: move this into body selector

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  height: 70px;
+  padding: 24px 0 23px 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (etojeDenys) [inline src/style.css:37]: move it to html selector

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+.search-bar__small .input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 19px;
+}
```
Comment (etojeDenys) [inline src/style.css:128]: you can combine some of these styles into one selector (`.input:focus`)

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+.search-bar__small .input {
+  width: 100%;
+  height: 42px;
+  padding: 9px 0 10px 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```
Comment (etojeDenys) [inline src/style.css:78]: same here, you can write almost all styles just into .input selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+.search-bar .search-input {
+  position: relative;
+}
+
+.search-bar .input {
+  width: 100%;
+  height: 70px;
+  padding: 24px 0 23px 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
```
Comment (etojeDenys) [inline src/style.css:31]: and move this color into a variable, so we can easily reuse her

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**